docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -305,25 +302,29 @@ By running
 
 ::
 
-    $ coala --bear-dirs=. --bears=HTMLTutorialBear -B
+    $ coala --bear-dirs=. --bears=PylintTutorialBear -B
 
 We can see that our Bear setting is documented properly. To use *coala*
-with our Bear on `sample.html` we run
+with our Bear on `sample.py` we run
 
 ::
 
-    $ coala --bear-dirs=. --bears=HTMLTutorialBear --files=sample.html
+    $ coala --bear-dirs=. --bears=PylintTutorialBear --files=sample.py
 
-To use our `htmllint_ignore` setting we can do
+To use our `pylint_rcfile` setting we can do
 
 ::
 
-    $ coala --bear-dirs=. --bears=HTMLTutorialBear \
-    > -S htmllint_ignore=optional_tag --files=sample.html
-
-This will not output anything because all the messages had the
-`optional_tag`.
+    $ coala --bear-dirs=. --bears=PythonTutorialBear \
+    > -S rcfile=my_rcfile --files=sample.py
 
 You now know how to write a linter Bear and also how to use it in your
 project.
+
 Congratulations!
+
+Where to find more...
+---------------------
+
+If you need more information about the ``@linter`` decorator, refer to the API
+documentation. It's more versatile than described in this little tutorial.
```

Comment:
@sils1297 did I promise? :O
but right using the rc-file then is not the best then...
@SanketDG is that possible? would do that indeed^^